Mohammed/index.js

```diff
@@ -0,0 +1,131 @@
+// 1.
+lowerLimitNum = 50;
+upperLimitNum = 99;
+
+let firstNum = 10;
+let secondNum = 20;
+
+/* if(firstNum >= lowerLimitNum && firstNum <= upperLimitNum)
+{
+    console.log(firstNum >= lowerLimitNum && firstNum <= upperLimitNum);
+    if(secondNum >= lowerLimitNum && secondNum <= upperLimitNum)
+    {
+        console.log(`The numbers ${firstNum} and ${secondNum} are in the range between ${lowerLimitNum} and ${upperLimitNum}`);
+    }
+}   */
+
+if ( ((firstNum >= lowerLimitNum) && (firstNum <= upperLimitNum)) &&    
+   ((secondNum >= lowerLimitNum) && (secondNum <= upperLimitNum)) )
+{
+    console.log(`The numbers ${firstNum} and ${secondNum} are in the range between ${lowerLimitNum} and ${upperLimitNum} ==> ${((firstNum >= lowerLimitNum) && (firstNum <= upperLimitNum)) &&    
+    ((secondNum >= lowerLimitNum) && (secondNum <= upperLimitNum))} `);
+}
+else
+{
+    console.log( `The numbers ${firstNum} and ${secondNum} are in the range between ${lowerLimitNum} and ${upperLimitNum} ==> ${((firstNum >= lowerLimitNum) && (firstNum <= upperLimitNum)) &&    
+    ((secondNum >= lowerLimitNum) && (secondNum <= upperLimitNum))} ` );
+}
+
+// 2-
+if ( ((firstNum >= lowerLimitNum) && (firstNum <= upperLimitNum)) ||    
```

Comment:
Here you have to make also a new variable wchich contains an integer number and check if it value is in the range of 50 and 99